utils/python_utils.py

```diff
@@ -14,14 +17,50 @@
         "pytorch_url": "cp311",
     },
 }
+REPO_DIR = Path(__file__).parent.parent
 
 
 def create_conda_env(pyver: str, name: str):
     command = ["conda", "create", "-n", name, "-y", f"python={pyver}"]
     subprocess.check_call(command)
 
 
+def pip_install_requirements(requirements_txt="requirements.txt",
+                            continue_on_fail=False,
+                            no_build_isolation=False,
+                            extra_args: Optional[List[str]]=None):
+    import sys
+    constraints_file = REPO_DIR.joinpath("build", "constraints.txt")
+    if not constraints_file.exists():
+        warnings.warn("constraints.txt could not be found, please rerun install.py.")
```

Comment:
My read here is that `build/constraints.txt` is an optional constraints file that TorchBench users could provide.  But the warning here seems to says that the file is a requirement?

Reply:
Yes, it is an optional constraint. We could update the warning message to sth like:

The `build/constrants.txt` file is not found. Please consider rerunning the `install.py` script to generate it.
It is recommended to have the `build/constrants.txt` file to prevent unexpected version change of `numpy` or `torch`.

Reply:
Ah ok, I see the `generate_pkg_constraints` func above now